### ISSUE TITLE
No Such command present in SPO Management shell

### ISSUE THREAD

**Issue by mganeshphani** [MicrosoftDocs/office-docs-powershell#1060]
not able to look for this command in SPO Management shell... is it worth to show in this section?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: ec45dc6b-ef71-3edf-78ba-ec1eb31bc260
* Version Independent ID: 7e6269f4-55de-c18b-82ad-b0e1306ad715
* Content: [Test-SPO365LinkSettings (sharepoint-online)](https://docs.microsoft.com/en-us/powershell/module/sharepoint-online/test-spo365linksettings?view=sharepoint-ps)
* Content Source: [sharepoint/sharepoint-ps/sharepoint-online/Test-SPO365LinkSettings.md](https://github.com/MicrosoftDocs/office-docs-powershell/blob/master/sharepoint/sharepoint-ps/sharepoint-online/Test-SPO365LinkSettings.md)
* Service: **sharepoint-powershell**
* Product: **unspecified**
* GitHub Login: @techwriter40
* Microsoft Alias: **kirks**

**Comment by Techwriter40** [comment]
@VesaJuvonen for your review. thanks

**Comment by tseward** [comment]
@Techwriter40 These cmdlets are actually yours :) They're on-prem cmdlets included in 2013 and 2016. They need to be moved over to on-prem docs and removed from SPO docs.

>man *-spo365*
Name                              Category  Module                    Synopsis
----                              --------  ------                    --------
Get-SPO365LinkSettings            Cmdlet    Microsoft.SharePoint.P... ...
Set-SPO365LinkSettings            Cmdlet    Microsoft.SharePoint.P... ...
Test-SPO365LinkSettings           Cmdlet    Microsoft.SharePoint.P... ...

**Comment by AndreaBarr** [comment]
@mganeshphani Thank you for this feedback

I want to make sure this gets investigated and updated so I am sending it over to the PowerShell team. 

I was able to get this into the backlog for the PowerShell team. The team will get this prioritized into a sprint plan. Thank you for reporting and making the docs better. Much appreciated. 
 
I made a note to request the team to update this when the work is complete.
